vizzy_gazebo/launch/gazebo.launch

```diff
@@ -1,63 +1,33 @@
 <?xml version="1.0"?>
 <launch>
 
-  <!-- launch the simulated world -->
-  <include file="$(find vizzy_gazebo)/launch/vizzy_sim_world.launch" />
-
-  <!-- launch the map server for the simulated world -->
-  <include file="$(find vizzy_gazebo)/launch/vizzy_sim_world_map_server.launch" />
-
   <!-- these are the arguments you can pass this launch file, for example paused:=true -->
+	<arg name="robot" default="vizzy"/> <!-- full, lower_body, foot -->
+  <arg name="pose" default="-x 0.0 -y 0.0 -z 0.2 -R 0.0 -P 0.0 -Y 1.57"/>
   <arg name="paused" default="false"/>
   <arg name="use_sim_time" default="true"/>
   <arg name="gui" default="true"/>
-  <arg name="headless" default="false"/>
-  <arg name="debug" default="false"/>
-  <arg name="world" default="gazebo" />
-  <arg name="map_frame" default="map" />
-  <arg name="map_service" default="/static_map" />
-  <arg name="map_topic" default="/map" />
-  <arg name="tf_prefix" default="" />
-  <arg name="x" default="0.0"/>
-  <arg name="y" default="0.0"/>
-  <arg name="z" default="0.2"/>
-  <arg name="yaw" default="1.57"/>
-  
-  <param name="tf_prefix" value="$(arg tf_prefix)" />
-  <arg name="base_frame_id" default="$(arg tf_prefix)/base_footprint" />
-  <arg name="odom_frame_id" default="$(arg tf_prefix)/odom" />
+  <arg name="world" default="vizzy_test_world.world" />
 
-  <!-- Load the URDF into the ROS Parameter Server -->
-  <param command="$(find xacro)/xacro.py '$(find vizzy_description)/robots/vizzy-with-collisions.urdf.xacro'" name="robot_description" />
-	 
-  <!-- launch the tf tree from the urdf (for all fixed joints + the hokuyo internal link) -->
-  <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher">
-    <param name="publish_frequency" type="double" value="5.0" />
-  </node>
+	<env name="GAZEBO_MODEL_PATH" value="$(find vizzy_gazebo)/media:$(optenv GAZEBO_MODEL_PATH)"/>
```

Comment:
This won't be an issue if we are using gazebo. In case we are using the real robot, we won't need the plugins. Therefore I suppose we can leave it as it is. The user can always specify the environment vraible in the .bashrc.
However, i have the feeling that this is not the proper place for the plugins. Have you seen some previous example?